common/compositor/va/varenderer.cpp

```diff
@@ -294,6 +367,67 @@ bool VARenderer::Draw(const MediaState& state, NativeSurface* surface) {
   param.filters = nullptr;
   param.filter_flags = VA_FRAME_PICTURE;
 
+  if(caps[0].type == VAProcColorBalanceNone) {
```

Comment:
please disable this feature by default until we have related interface to pass the settings from app to hwc.